components/network/openvpn/Makefile

```diff
@@ -15,15 +15,16 @@
 # Copyright 2022 Niklas Poslovski
 #
 
-USE_OPENSSL11=	yes
+OPENSSL_VERSION= 3.1
```

Comment:
You do not need to set this.  It is default.  If you omit such lines, then future component updates/rebuilds will be easier.